src/main/java/github/weichware10/util/data/TrialData.java

```diff
@@ -22,44 +34,67 @@
     private List<DataPoint> dataPoints;
 
     /**
-     * Stores the TrialData for the different tools internally.
-     *
-     * @param toolType - the tooltype of the stored data
-     * @param trialId  - the id of the trial
-     * @param configId - the configuration of the stored data
+     * Konstruktor für Jackson.
```

Comment:
Wirklich Jackson ?

Reply:
Jackson ist der Name des Moduls, welches Instanzen von Klassen in .json umwandelt bzw. .json in Instanzen umwandelt.